lib/shared/bucketing-assembly-script/assembly/testHelpers.ts

```diff
@@ -148,6 +148,18 @@ export function testBucketedUserConfigClass(userConfigStr: string): string {
     return userConfig.stringify()
 }
 
+export function echoString(str: string): string {
+    return str
+}
+
+export function echoUint8Array(str: Uint8Array): Uint8Array {
```

Comment:
minor nitpick maybe call the variable something other than 'str' here since you use that above.  Maybe something else like 'data'?